src/main/java/bio/terra/common/opentelemetry/OpenTelemetryConfig.java

```diff
@@ -52,7 +57,7 @@ public SdkTracerProvider terraTraceProvider(
     spanProcessors.stream().forEach(tracerProviderBuilder::addSpanProcessor);
     tracerProviderBuilder.setSampler(
         new ExcludingUrlSampler(
-            tracingProperties.excludedUrls(),
+            Optional.ofNullable(tracingProperties.excludedUrls()).orElse(DEFAULT_EXCLUDED_URLS),
```

Comment:
Using `Optional.of` creates a new object on the heap, you can avoid that by using `Objects.requireNonNullElse()`
```suggestion
            Objects.requireNonNullElse(tracingProperties.excludedUrls(), DEFAULT_EXCLUDED_URLS),
```